presto-docs/src/main/sphinx/release/release-0.177.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.177
+=============
+
+General
+-------
+
+* Fix correctness issue when performing range comparisons over columns of type ``CHAR``.
+* Fix correctness issue due to mishandling of nulls and non-deterministic expressions in
```

Comment:
The double negative makes it harder to read: "unless ... is disabled". "when ... is enabled" is clearer, IMO.